cmd/up/exec.go

```diff
@@ -242,14 +242,27 @@ func (eg *envsGetter) getEnvs(ctx context.Context) ([]string, error) {
 		envs = append(envs, fmt.Sprintf("%s=%s", env.Name, env.Value))
 	}
 
-	pathValue := eg.pathGetter("PATH")
-	if pathValue != "" {
-		envs = append(envs, fmt.Sprintf("PATH=%s", pathValue))
-	}
+	envs = append(envs, eg.getDefaultLocalEnvs()...)
 
 	return envs, nil
 }
 
+func getDefaultLocalEnvs() []string {
+	var envs []string
+
+	path := os.Getenv("PATH")
+	if path != "" {
+		envs = append(envs, fmt.Sprintf("PATH=%s", path))
+	}
+
+	term := os.Getenv("TERM")
```

Comment:
@AdrianPedriza is `TERM` enough to fix colors, etc?

I ask because in other projects I've seen both `TERM` and `COLORTERM` are used for this